### PR TITLE
Make the proxy functionality optional

### PR DESCRIPTION
**Pull request by annando**
The "proxy" functionality is now mostly replaced by another functionality. It is only used for rare cased like pictures that had been fetched via OEmbed and a few others.

Since this functionality can be used for redirecting any links we can now disable it.

### CODE REVIEW

File: src/Module/Admin/Site.php [hunk 1]
```diff
@@ -551,6 +553,7 @@ public static function content(array $parameters = [])
...
 			'$explicit_content'       => ['explicit_content', DI::l10n()->t('Explicit Content'), DI::config()->get('system', 'explicit_content'), DI::l10n()->t('Set this to announce that your node is used mostly for explicit content that might not be suited for minors. This information will be published in the node information and might be used, e.g. by the global directory, to filter your node from listings of nodes to join. Additionally a note about this will be shown at the user registration page.')],
+			'$proxify_content'        => ['proxify_content', DI::l10n()->t('Proxify external content'), DI::config()->get('system', 'proxify_content'), DI::l10n()->t('Redirect external content via the proxy functionality. This is used for example for some OEmbed accesses and in some other rare cases.')],
```
Comment (MrPetovan) [inline src/Module/Admin/Site.php]: As far as I know, the proxy doesn't redirect, it makes a local copy of the remote resource that is then accessed instead.

Comment (annando) [reply src/Module/Admin/Site.php]: Well, that's not completely true: https://github.com/friendica/friendica/blob/2021.09-rc/src/Module/Proxy.php#L69-L72

Comment (MrPetovan) [reply src/Module/Admin/Site.php]: Indeed, but it should be clearer in the description.

Comment (annando) [reply src/Module/Admin/Site.php]: I changed a word.

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
It doesn't say what the proxy does but at least it isn't partially wrong anymore!

**Comment by annando** [comment]
That was my intention. And I'm proud having typed this in the deepest night :-)

**Pull request merged**